layouts/partials/templates/blocks.html

```diff
@@ -14,8 +14,10 @@
 {{ $header := . | replaceRE "</?h2.*?>" "" | htmlUnescape | safeHTML }}
 <li><a href="#{{ $id }}">{{ $header }}</a></li>
 {{ end }}
+{{ if  $section }}
 {{ $.ctx.Scratch.Add "sections" $section }}
 {{ end }}
+{{ end }}
```

Comment:
Can you explain this change @vitalyzhakov ? It seems possibly unrelated. Maybe it should go into a separate pull request.

Reply:
Yes.
Before changes I see error
```
6:23:21 PM: Error: Error building site: failed to render pages: render of "page" failed: execute of template failed: template: docs/single.html:3:3: executing "content" at <partial "docs/content-page" (dict "ctx" . "page" .)>: error calling partial: execute of template failed: template: partials/docs/content-page.html:12:3: executing "partials/docs/content-page.html" at <partial . $>: error calling partial: execute of template failed: template: partials/templates/task.html:11:3: executing "partials/templates/task.html" at <partial "templates/blocks" .>: error calling partial: "/opt/build/repo/layouts/partials/templates/blocks.html:17:4": execute of template failed: template: partials/templates/blocks.html:17:4: executing "partials/templates/blocks.html" at <$.ctx.Scratch.Add>: error calling Add: reflect: call of reflect.Value.Type on zero Value
```
After changes I see result, what helps me to debug mistakes with including code.

Do we really need separate MR for 2 lines of code?